guidelines.md

```diff
@@ -219,6 +220,19 @@ There are different techniques that can be used such as **Daki Exercise**, **Sto
 - What was bad
 - What can improve
 
+### Deployment
+
+It is proven that a high deploy frequency is a good DevOps practice and gives a lot of benefits in the Agile process. Google considers multiple deploys per day as a [factor to consider a team as an Elite Performer](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling.).
+
+Deploying more frequently:
+ - Represents having smaller changes, which means less risk.
+ - Decreases the time between a feature being ready and being live.
+ - Encourages better feature QA, since features will spend less time on Staging until they launch to Production.
+ - Increases code review quality, since changes tend to be smaller.
+ - Incentivizes breaking up bigger tasks in smaller ones that can be deployed as soon as they are ready.
```

Comment:
More benefits:

- Facilitates bugs detection.
- ... not sure how to phrase it here, but something that mentions it's best for the team, as they don't have to meet a single deploy day deadline.